### DISCUSSION TITLE
github tecnologia

### DISCUSSION THREAD

**Discussion by stxgho** [community/community#69073] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

hola comunidad, la pagina de github de que esta hecho? o sea que tecnologia usa?

**Comment by 3LIAS-MB** [comment]
Hola, GitHub está construido principalmente utilizando Ruby on Rails y Git como su sistema de control de versiones, con Erlang para la capa de red y PostgreSQL como su sistema de gestión de bases de datos. Además, utiliza Redis para el almacenamiento en caché, JavaScript (tanto en el lado del cliente como en el servidor), GraphQL como lenguaje de consulta, Docker para la contenerización, Amazon Web Services (AWS) para la infraestructura, y varias herramientas de desarrollo y despliegue como GitHub Actions, Travis CI, Sentry, entre otras, para brindar una plataforma de desarrollo colaborativo y control de versiones altamente eficiente y escalable.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
